src/engine/strat_engine/filesystem.rs

```diff
@@ -5,15 +5,20 @@
 use std::path::{Path, PathBuf};
 use std::process::Command;
 
-use devicemapper::DM;
-use devicemapper::{ThinDev, ThinDevId, ThinStatus, ThinPoolDev};
+use devicemapper::{Bytes, DM, Sectors, ThinDev, ThinDevId, ThinStatus, ThinPoolDev};
+use devicemapper::consts::{IEC, SECTOR_SIZE};
+
+use nix::sys::statvfs::statvfs;
+use nix::sys::statvfs::vfs::Statvfs;
 
 use super::super::engine::{Filesystem, HasName, HasUuid};
 use super::super::errors::{EngineError, EngineResult, ErrorEnum};
 use super::super::types::FilesystemUuid;
 
 use super::serde_structs::{FilesystemSave, Recordable};
 
+pub const FILESYSTEM_LOWATER: Sectors = Sectors(256 * IEC::Mi / (SECTOR_SIZE as u64)); // = 256 MiB
```

Comment:
We discussed this value in the meeting, right?  I'm not sure it is correct.  It is just a first guess.